Assets/Samples/Continuous/ContinuousDemo.cs

```diff
@@ -23,6 +30,8 @@ public class ContinuousDemo : MonoBehaviour {
 		BarcodeScanner.OnReady += (sender, arg) => {
 			// Set Orientation & Texture
 			Image.transform.localEulerAngles = new Vector3(0f, 0f, BarcodeScanner.Camera.GetRotation());
+			float scaleY = BarcodeScanner.Camera.IsVerticalyMirrored() ? -1f : 1f;
```

Comment:
I think this can belong to the camera itself, no?
Something like ICamera.TextureParameters -> rotation + scale
What do you think?